lms/static/js/learner_dashboard/views/program_details_sidebar_view.js

```diff
@@ -0,0 +1,47 @@
+(function(define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'js/learner_dashboard/views/explore_new_programs_view',
+            'js/learner_dashboard/views/certificate_view',
+            'text!../../../templates/learner_dashboard/sidebar.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             NewProgramsView,
+             CertificateView,
+             sidebarTpl
+         ) {
+             return Backbone.View.extend({
+                 el: '.sidebar',
+
+                 tpl: _.template(sidebarTpl),
+
+                 initialize: function(data) {
+                     this.context = data.context;
+                 },
+
+                 render: function() {
+                     this.$el.html(this.tpl(this.context));
+                     this.postRender();
+                 },
+
+                 postRender: function() {
+                     this.newProgramsView = new NewProgramsView({
```

Comment:
I didn't write this, I just copied this from sidebar_view.js  
I'm assuming all of this rewritten, but i figured it would be helpful to start with some of the current code to simplify the process of changing it